api/src/Responder/Contract/ResponderInterface.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: redjik
+ * Date: 16.07.15
+ * Time: 0:20
+ */
+
+namespace Spira\Responder\Contract;
+
+use Symfony\Component\HttpFoundation\Response;
+
+interface ResponderInterface
+{
+    /**
+     * @return Response
+     */
+    public function getResponse();
+
+    /**
+     * Return an error response.
+     *
+     * @param string $message
+     * @param int  $statusCode
+     *
+     * @throws \Symfony\Component\HttpKernel\Exception\HttpException
+     *
+     */
+    public function error($message, $statusCode);
```

Comment:
I would prefer that we don't have any of these error methods, as we already handle errors with exceptions. We should not have two methods of returning errors.